src/FluentValidation.DependencyInjectionExtensions/ServiceCollectionExtensions.cs

```diff
@@ -23,17 +23,41 @@ namespace FluentValidation {
 	using System.Reflection;
 
 	public static class ServiceCollectionExtensions	{
+
 		/// <summary>
 		/// Adds all validators in specified assemblies
 		/// </summary>
 		/// <param name="services">The collection of services</param>
 		/// <param name="assemblies">The assemblies to scan</param>
-		/// <param name="lifetime">The lifetime of the validators. The default is transient</param>
-		/// <param name="filter">Optional filter that allows certain types to be skipped from registration.</param>
+		/// <param name="configExpression">Optional config object that contains factory registration information.</param>
 		/// <returns></returns>
-		public static IServiceCollection AddValidatorsFromAssemblies(this IServiceCollection services, IEnumerable<Assembly> assemblies, ServiceLifetime lifetime = ServiceLifetime.Transient, Func<AssemblyScanner.AssemblyScanResult, bool> filter = null) {
+		public static IServiceCollection AddValidatorsFromAssemblies(this IServiceCollection services, IEnumerable<Assembly> assemblies, Action<FluentValidationDiConfiguration> configExpression = null) {
+			var config = new FluentValidationDiConfiguration(ValidatorOptions.Global);
+			configExpression?.Invoke(config);
+			return services.AddValidatorsFromAssemblies(assemblies, config);
+		}
+
+		/// <summary>
+		/// Adds all validators in specified assemblies
+		/// </summary>
+		/// <param name="services">The collection of services</param>
+		/// <param name="assemblies">The assemblies to scan</param>
+		/// <param name="config">Optional config object that contains factory registration information.</param>
+		/// <returns></returns>
+		public static IServiceCollection AddValidatorsFromAssemblies(this IServiceCollection services, IEnumerable<Assembly> assemblies, FluentValidationDiConfiguration config = null) {

```

Comment:
Please make this method internal - the other overload which takes the `configExpression` is the one that consumers should be using.